packages/lisk-cryptography/test/keys.ts

```diff
@@ -133,4 +134,12 @@ describe('keys', () => {
 			);
 		});
 	});
+
+	describe('#getAddressFromPrivateKey', () => {
+		it('should create correct address', () => {
+			return expect(
+				getAddressFromPrivateKey(defaultPrivateKey.slice(0, 64)),
```

Comment:
I think the same test, but with 32 byte input, would be good. Or do you think its a bit of overdoing since both cases or covered with tests for getPublicKey?

Reply:
yea i think it's covered by the getPublicKey, so not needed here

Reply:
ok